package.json

```diff
@@ -264,9 +264,6 @@
     "@types/react-sortable-hoc": "^0.7.1",
     "@types/react-window": "^1.8.2",
     "@types/redux-logger": "^3.0.7",
-    "@types/storybook__addon-centered": "^3.3.2",
-    "@types/storybook__addon-knobs": "^5.0.2",
-    "@types/storybook__react": "^4.0.2",
```

Comment:
note for reviewer: i don't think these @types/storybook deps are needed anymore (https://github.com/storybookjs/storybook/issues/8233#issuecomment-543538702)